flink-metrics/flink-metrics-prometheus/src/main/java/org/apache/flink/metrics/prometheus/PrometheusPushGatewayReporterOptions.java

```diff
@@ -63,4 +63,14 @@
 				" Before disabling this option please ensure that your" +
 				" label values meet the %s.", LinkElement.link("https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels", "Prometheus requirements"))
 			.build());
+
+	public static final ConfigOption<String> GROUPING_KEY = ConfigOptions
+		.key("groupingKey")
+		.defaultValue("")
+			.withDescription(Description.builder()
+				.text("Specifies the grouping key which is the group and global labels of all metrics." +
+					" The label name and value is separated with '=', and labels are separated with ';', e.g. k1=v1;k2=v2." +
```

Comment:
```suggestion
					" The label name and value are separated by '=', labels are separated by ';', e.g., k1=v1;k2=v2." +
```

Bonus points for replacing the example with a `code` element.

Reply:
Do you mean e.g., < code > k1=v1;k2=v2 < /code >. ? I found the generated doc display it as plain text. Or if I have a mistake with it?

Reply:
You have to use `TextElement#code` for that, works just like adding a link.